backend/src/main/kotlin/app/ehrenamtskarte/backend/common/utils/DevLogger.kt

```diff
@@ -0,0 +1,28 @@
+package app.ehrenamtskarte.backend.common.utils
+import org.slf4j.Logger
+
+fun isDevEnv(): Boolean = Environment.getVariable("APP_ENV") == "dev"
```

Comment:
there is a standard way to manage different levels of logging in slf4j per environment.
we could add configuration and log "info" level for prod and "debug" for dev, for example

Reply:
Thanks!